### PR TITLE
add view podpreset/sc resources to view role

### PR DESCRIPTION
**Pull request by bparees**

### CODE REVIEW

**Comment by bparees** [comment]
@deads2k ptal

File: pkg/bootstrap/docker/openshift/servicecatalog.go [hunk 1]
```diff
@@ -35,11 +35,37 @@ func (h *Helper) InstallServiceCatalog(f *clientcmd.Factory, publicMaster, catal
 		return errors.NewError("cannot obtain API clients").WithCause(err).WithDetails(h.OriginLog())
 	}
 
-	// Grant all users with the edit role, the ability to manage service catalog instance/binding resources.
+	// Grant all users with the view role, the ability to view service catalog instance/binding resources and
+	// podpreset resources.
+	scViewRule, err := authzapi.NewRule("get", "list", "watch").Groups("servicecatalog.k8s.io").Resources("instances", "bindings").Rule()
```
Comment (deads2k) [inline pkg/bootstrap/docker/openshift/servicecatalog.go]: `instances.servicecatalog.k8s.io` has a `spec.parameters` field which appears likely to hold secrets entered by the user.  `view` is a non-escalating role`, so granting `view` to a user shouldn't allow them to get at your secret information.

**Comment by bparees** [comment]
the console is going to work around not being able to view these resources, closing the PR.

**Comment by bparees** [comment]
thanks for the catch, @deads2k

**Pull request closed without merging**